app/src/main/java/co/nimblehq/compose/crypto/ui/common/AppDialogPopUp.kt

```diff
@@ -0,0 +1,78 @@
+package co.nimblehq.compose.crypto.ui.common
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.window.Dialog
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.ui.theme.*
+
+@Composable
+fun AppDialogPopUp(
+    onDismiss: () -> Unit,
+    onClick: () -> Unit,
+    @StringRes title: Int,
+    @StringRes message: Int,
+    @StringRes actionText: Int,
+) {
+    Dialog(
+        onDismissRequest = onDismiss
+    ) {
+        Column(
+            modifier = Modifier
+                .background(Color.White)
```

Comment:
Because the app supports light/dark themes. I'd suggest having the dialog in both theme for better UX 🌔